packages/opentelemetry-core/src/platform/browser/environment.ts

```diff
@@ -25,7 +25,9 @@ import {
  * Gets the environment variables
  */
 export function getEnv(): Required<ENVIRONMENT> {
-  const _window = window as typeof window & RAW_ENVIRONMENT;
+  const _window = (typeof window === 'undefined'
```

Comment:
@obecny is there some reason not to use `globalThis` or similar? Maybe from core we should export something like

```
export const getGlobal = function () {
  if (typeof globalThis !== 'undefined') { return globalThis; }
  if (typeof self !== 'undefined') { return self; }
  if (typeof window !== 'undefined') { return window; }
  if (typeof global !== 'undefined') { return global; }
  throw new Error('unable to locate global object');
};
```

Reply:
yah I think this type of check is a good idea.  I didn't use `globalThis` because I wasn't sure if the browser support was there or not.

I can add a `getGlobal` function to core.  Do you see any other places it's needed besides https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-core/src/platform/node/environment.ts#L29?

Reply:
>I wasn't sure if the browser support was there or not.

It's not there in IE at least. I would wait on feedback from @obecny before making changes to core

Reply:
tbh I don't remember why it is not using global.
@ryhinchey there is already a property for that, but not in the core but in api `_globalThis` so just use this one.
If there is anything that needs to be changed in this `_globalThis` then feel free to open PR in `api`

Reply:
here -> https://github.com/open-telemetry/opentelemetry-js-api/blob/main/src/platform/browser/globalThis.ts

Reply:
@obecny sounds good - thanks

Reply:
now that I think about it, I'll break out the service worker support into a separate PR and keep this one just to server side rendering